docs/implementation-references/getting-started-guide.md

```diff
@@ -283,7 +308,7 @@ Wait for TO2 protocol completed message and Device is onboarded Successfully.
 Expect the following message on successful TO2 completion.
 
 ```
-TO2 complete, GUID is d43c6dc6...
+TO2 completed successfully.
```

Comment:
Aren't we adding the GUID here? I thought that was one of the changes requested as part of automation

Reply:
Not part of the latest TO2 completion logging.